analyzer/src/main/kotlin/curation/ClearlyDefinedPackageCurationProvider.kt

```diff
@@ -172,6 +172,6 @@ class ClearlyDefinedPackageCurationProvider(
             }
         }
 
-        return curations
+        return curations.mapValues { (_, curations) -> curations.distinct() }
```

Comment:
I'm wondering whether we should instead extend the check inline 166 / insert a new check between lines 166 and 167 to not add the `PackageCuration` if it's already present. But it's a bit tricky due to the `data.copy(comment = "Provided by ClearlyDefined.")`. *Could* be something like

                if (data != PackageCurationData()) {
                    val curation = PackageCuration(
                        id = pkgId,
                        data = data.copy(comment = "Provided by ClearlyDefined.")
                    )

                    curations.getOrPut(pkgId) { mutableListOf() }.also { if (curation !in it) it += curation }
                }

But your variant is probably more efficient.

Reply:
Another option would be to use a set for the curations and convert it to a list in the end, but I think with regards to efficiency this should be about the same as filtering duplicates at the end.